samples/ControlCatalog/ControlCatalog.csproj

```diff
@@ -23,6 +23,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\packages\Avalonia\Avalonia.csproj" />
+    <ProjectReference Include="..\..\src\Avalonia.Controls.ColorPicker\Avalonia.Controls.ColorPicker.csproj" />
```

Comment:
@maxkatz6 Here is a strange one. This is the last error: "...\Avalonia\samples\ControlCatalog.NetCore\ControlCatalog.NetCore.csproj : error NU1104: Unable to find project 'C:\Users\Dean\Desktop\Software\Avalonia\src\Avalonia.Controls.DataGrid\Avalonia.Controls.ColorPicker.csproj'. Check that the project reference is valid and that the project file exists."

However, this reference is good. Not sure what is happening for the build process to be looking in the wrong directory.

Reply:
Cleared all caches, deleted `.vs` folder and same error persists. Will do more internet searches but this is quite strange. Will probably take another look tomorrow.

Reply:
It passed all build checks now. Definitely a local issue and not a problem with the code.

Reply:
The error was actually correct -- just indicating the wrong project. It was an error occurring in LeakTests and Diagnostics projects which are referenced from ControlCatalog.